packages/backend-function/API.md

```diff
@@ -31,6 +31,7 @@ export type FunctionProps = {
     environment?: Record<string, string | BackendSecret>;
     runtime?: NodeVersion;
     schedule?: FunctionSchedule | FunctionSchedule[];
+    minify?: boolean;

```

Comment:
Could you please create `FunctionBundlingOptions` and nest `minify` under it?

We'll be exposing more bundling options in the future so let's future proof it.

Reply:
@sobolk
Thank you for your review and suggestion.
I have modified it as you suggested.